### DISCUSSION TITLE
How to run multiple commands

### DISCUSSION THREAD

**Discussion by AsuraKev** [oclif/oclif#1512] [Q&A]
Hi there!

I am new to Ocliff and currently evaluating the tools! I see that there are options to define multiple commands in the solution, is there a way to chain the commands together when users call the CLI?

thnks!

**Comment by mdonnalley** [comment]
Users can pipe input from one command to the next. 

So, for example, `cmd1` prints a string to the terminal and `cmd2` takes an argument:

```
cmd1 | cmd2
```

as opposed to

```
$ cmd1
foobar

$ cmd2 foobar
```
